src/index.json

```diff
@@ -975,6 +975,53 @@
                     "version": "0.1.0"
                 }
             }
+        ],
+        "hanaonazure": [
+            {
+                "filename": "hanaonazure-0.1.5-py2.py3-none-any.whl",
+                "sha256Digest": "0fb0aec0a18997877639d2674ff2cd908def9d0ff0e356f18c2067d1c7a07d76",
+                "downloadUrl": "https://github.com/Azure/azure-hanaonazure-cli-extension/releases/download/0.1.5/hanaonazure-0.1.5-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.0.25",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
+                        "Programming Language :: Python :: 2.7",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
+                                    "name": "Microsoft Corporation",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-hanaonazure-cli-extension"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.29.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "hanaonazure",
+                    "summary": "Microsoft Azure Command-Line Tools SAP HANA Command Module",
```

Comment:
Change this to a summary that does not refer to the CLI/Azure as both are implied. I believe the summary in your setup.py on your repo is "Additional commands for working with SAP HanaOnAzure instances." which fits our summary guidelines. Both summaries will need to match.

Reply:
changed